wsd/RequestDetails.cpp

```diff
@@ -152,7 +152,8 @@ RequestDetails::RequestDetails(const std::string& wopiSrc, const std::vector<std
     processURI();
 }
 
-RequestDetails::Method RequestDetails::stringToMethod(std::string const & method) {
+RequestDetails::Method RequestDetails::stringToMethod(const std::string_view method)
```

Comment:
It's const std::string_view here in the .cpp but std::string_view in the .hpp below

Reply:
> It's const std::string_view here in the .cpp but std::string_view in the .hpp below

Yes, that's intentional. The pattern is that marking a value-type as const in the declaration doesn't add value and can be misleading (i.e. could be read as `const&`), but that doesn't stop one from marking it as const in the definition to prevent accidental modifications.

In other words, constness of a by-value parameter is immaterial distraction, but the implementation can benefit from it as an implementation-detail. Clang-tidy recommends this as well.